client/blocks/plugins-update-manager/schedule-form.tsx

```diff
@@ -36,8 +36,12 @@ interface Props {
 }
 export const ScheduleForm = ( props: Props ) => {
 	const { siteSlug } = props;
-	const { data: dataPlugins } = useSitePluginsQuery( siteSlug );
-	const { plugins = [] } = dataPlugins ?? {};
+	const { data } = useCorePluginsQuery( siteSlug );
+	const plugins = useMemo( () => {
+		const plugins = data ?? [];
+		// don't display managed plugins
+		return plugins.filter( ( plugin ) => ! plugin.is_managed );
+	}, [ data ] );
```

Comment:
I suggest moving this filtering part directly inside the `useCorePluginsQuery` hook.
You can provide one more param next to the `siteSlug`, something like "hideManagedPlugins".

I'm suggesting it because there will be two more components that use the same hook. So, let's avoid duplicating that filtering piece of code.

Reply:
Thank you for the suggestion. Done in https://github.com/Automattic/wp-calypso/pull/88134/commits/1a83e4f8d71c2d85cc865e42c21af6b498e76b16